kinder/cmd/kinder/build/baseimage/baseimage.go

```diff
@@ -67,14 +66,7 @@ func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
 	switch strings.ToLower(flags.CRI) {
 	case "containerd":
 		// Use build base image from Kind
-		ctx := kindbase.NewBuildContext(
-			kindbase.WithImage(flags.Image),
-			kindbase.WithSourceDir(flags.Source),
-		)
-		if err := ctx.Build(); err != nil {
-			return errors.Wrap(err, "build failed")
-		}
-		return nil
+		return errors.New("Kinder doesn't support building base images for conainerd. Use a base image from Kind")
```

Comment:
error should start lowercase
i though golint CI should be capturing these.

Reply:
agree...